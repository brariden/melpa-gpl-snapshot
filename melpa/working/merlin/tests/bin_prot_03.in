#!/bin/bash
. functions.sh || exit 1

reset bin_prot_03.ml
package bin_prot
tell <<END
module Test_binprot = struct 
  type 'a t = { x : int ; y : 'a } with bin_io
end
END
cmd dump env
