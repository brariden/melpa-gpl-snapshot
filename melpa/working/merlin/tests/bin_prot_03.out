["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"result":true}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",[["File \"bin_prot_03.ml\", line 2, characters 7-46","module Test_binprot :\n  sig\n    type 'a t = { x : int; y : 'a; }\n    val bin_size_t : ('a -> int) -> 'a t -> int\n    val bin_write_t : 'a Bin_prot.Write.writer -> 'a t Bin_prot.Write.writer\n    val bin_writer_t :\n      'a Bin_prot.Type_class.writer -> 'a t Bin_prot.Type_class.writer\n    val bin_read_t : 'a Bin_prot.Read.reader -> 'a t Bin_prot.Read.reader\n    val __bin_read_t__ :\n      'a Bin_prot.Read.reader -> (int -> 'a t) Bin_prot.Read.reader\n    val bin_reader_t :\n      'a Bin_prot.Type_class.reader -> 'a t Bin_prot.Type_class.reader\n    val bin_t : 'a Bin_prot.Type_class.t -> 'a t Bin_prot.Type_class.t\n  end"]]]
